### ISSUE TITLE
Is there International support yet?

### ISSUE THREAD

**Issue by Brandon1212b** [pigskin/kodi-gamepass#444]
I apologize if this is not the correct place to ask this. I saw years ago there was discussion about implementing game pass international support. Has this been added yet? I want to know if it would work outside of just game pass Europe before I purchase for the season. Thank you

**Comment by jm-duke** [comment]
Well, that's harder do answer than you'd expect ;-)

The NFL renamed and re-designed their services quite a bite in the last years. I did some research for #437 and it kinda seems that NFL Europe and NFL International were merged at some point in time. The original (US) Game Pass was renamed to NFL Game Pass Domestic, retired this year and replaced by NFL+.

Today, there is only one NFL Game Pass for non-US countries (https://support.nflgamepass.com/hc/en-gb/articles/360004691153). I have an account for this Game Pass offering and the add-on is working fine, but I live in Europe. I don't know if the service under the hood is using some other streaming provider in non-European countries.

But you have the option to try the add-on before you pay the Pro fee. If you sign-up for Pro, you enter a 7-day free trial period. You are free to cancel the subscription in this 7-day period and are not billed. Imo, enough time to test the Kodi add-on from your location and provide feedback here :)
